types/tar/index.d.ts

```diff
@@ -198,14 +201,17 @@ export const knownExtended: {
 export const headerSize: number;
 export const blockSize: number;
 
+/**
+ * A writable stream. Write tar data to it and it will emit entry events for each entry parsed from the tarball. This is used by tar.Extract.
+ */
+export const Parse: {
```

Comment:
The types here aren't right -- you're saying that `new Parse()` creates a `ParseStream`. But `ParseStream` is returned from `Extract()` and is a different implementation (the `Unpack` class in the source code) than the `Parse` class.